crypto/x509/x509_vfy.c

```diff
@@ -1782,13 +1782,19 @@ int X509_cmp_current_time(const ASN1_TIME *ctm)
     return X509_cmp_time(ctm, NULL);
 }
 
+static int ascii_isdigit(char inchar) {
+  if (inchar > 0x2F && inchar < 0x3A)
+    return 1;
+  return 0;
+}
```

Comment:
Since this is the second time you've defined this, I suggest moving it to ctype.c instead.

Reply:
Move there and added to ctype.h as well